src/BowlerServiceProvider.php

```diff
@@ -34,8 +35,11 @@ public function register()
         $rbmqPort = config('queue.connections.rabbitmq.port');
         $rbmqUsername = config('queue.connections.rabbitmq.username');
         $rbmqPassword = config('queue.connections.rabbitmq.password');
-        $this->app->bind(Connection::class, function () use ($rbmqHost, $rbmqPort, $rbmqUsername, $rbmqPassword) {
-            return new Connection($rbmqHost, $rbmqPort, $rbmqUsername, $rbmqPassword);
+        $rbmqConnectionTimeout = (int) config('queue.connections.rabbitmq.connection_timeout');
+        $rbmqReadWriteTimeout = (int) config('queue.connections.rabbitmq.read_write_timeout');
+        $rbmqHeartbeat = (int) config('queue.connections.rabbitmq.heartbeat');
```

Comment:
enforce the defaults from here, for if they're not set they'll be sent as empty and will override the default.